prep-javascript-loops/main.js

```diff
@@ -0,0 +1,68 @@
+// create your loops here.
+function whileLoop1() {
+  const numArray = [0, 1, 2, 3, 4, 5, 6, 7, 8];
+  let index = 0;
+  while (index < 9) index++;
```

Comment:
this loop is not currently working. 

Make sure to retest after clearing out the array above.